lib/private/Log.php

```diff
@@ -359,6 +359,7 @@ public function logData(string $message, array $data, array $context = []): void
 			$context['level'] = $level;
 		} catch (Throwable $e) {
 			// make sure we dont hard crash if logging fails
+			error_log('Error when trying to log exception: ' . $e->getMessage() . ' ' . $e->getTraceAsString());
```

Comment:
Isn't that enough? Otherwise we might risk that secrets get logged?

```suggestion
			error_log('Error when trying to log exception: ' . $e->getMessage());
```

Reply:
Well in general I would agree but only the exception message might not be enough to investigate what went wrong. And this handler should only be called as a last resort.

In my case the home directory of the user was missing so I'd only get something like "File not found". The stacktrace helped my finding the issue. But yeah if this is a security concern the message would be better than nothing :-)

Reply:
agreed, this is the error after an error, so fine to log with details.